ReactiveCocoaTests/Swift/ActionSpec.swift

```diff
@@ -176,12 +177,15 @@ class ActionSpec: QuickSpec {
 				let result = action.apply(0).first()
 				expect(result?.value) == 1
 				expect(values).toEventually(equal([ true, false, true ]))
+				
+				_ = cocoaAction
```

Comment:
That's to keep `cocoaAction` from deallocating before the end of the test—because when it deallocates, the producer will complete.